frontend/workflows/dynamodb/src/update-capacity.tsx

```diff
@@ -55,6 +56,18 @@ const TableDetails: React.FC<TableDetailsChild> = ({ enableOverride, notes = []
 
   const tableDetailsNotes = notes.filter(note => note.location === "table-details");
   const limitsNotes = notes.filter(note => note.location === "scaling-limits");
+  const getChipStatus = (status: IClutch.aws.dynamodb.v1.Table.Status) => {
+    switch (true) {
+      case status.toString() === "ACTIVE":
+        return "active";
+      case status.toString() === "UPDATING" || status.toString() === "CREATING":
+        return "pending";
+      case status.toString() === "DELETING":
+        return "warn";
+      default:
+        return "neutral";
+    }
+  };
```

Comment:
Would probably adjust this to something like below, makes it a bit easier to read and extend

```suggestion
    switch (status.toString()) {
      case "ACTIVE":
        return "active";
      case "UPDATING":
      case "CREATING":
        return "pending";
      case "DELETING":
        return "warn";
      default:
        return "neutral";
    }
  };
```